subsys/logging/log_output.c

```diff
@@ -522,6 +528,24 @@ void log_output_msg_process(const struct log_output *log_output,
 	log_output_flush(log_output);
 }
 
+void log_output_dropped_process(const struct log_output *log_output, u32_t cnt)
+{
+	char buf[3];
+	int len;
+	static const char prefix[] = DROPPED_COLOR_PREFIX "--- ";
+	static const char postfix[] =
+			" messages dropped ---\r\n" DROPPED_COLOR_POSTFIX;
+	log_output_func_t outf = log_output->func;
+	struct device *dev = (struct device *)log_output->control_block->ctx;
+
+	cnt = min(cnt, 99);
```

Comment:
But why? What's wrong with printing 5-digit number?

Reply:
increased to 4 digits. It's unlikely to see more than 9999 lost messages and even then exact number is not important.